bundles/org.openhab.binding.konnected/README.md

```diff
@@ -27,25 +27,14 @@ The blink setting will disable the transmission LED on the Konnected Alarm Panel
 
 
 ## Channels
-
-The auto discovered thing adds two default channels.
-
-| Channel | Channel Id | Channel Type | Description                                              |
-|---------|------------|--------------|----------------------------------------------------------|
-| 1       | Zone_6     | Switch       | A Switch channel for zone 6                              |
-| 2       | Out        | Actuator     | The Channel for the Out Pin on the Konnected Alarm Panel |
-
-One channel for Zone 6 which is a sensor type channel, and one channel for the out pin that is an actuator type channel.
-These channels represent the two pins on the Konnected Alarm Panel whose type cannot be changed.
-For zones 1-5, you will need to add channels for the remaining zones that you have connected and configure them with the appropriate configuration parameters for each channel.
-
+You will need to add channels for the zones that you have connected and configure them with the appropriate configuration parameters for each channel.
```

Comment:
Channels or items? Also, please add new line after header.

Reply:
Channels. The big difference between this binding and others that I've used is that channels has to be added and configured. I'm not sure if there is another way of doing it, but this makes sense. This design has not changed from the original wifi only version of the binding.

The original binding (only wifi) had 2 channels preconfigured, but allowed you to add more channels for the same zone. You may want to do that if you have a buzzer connected, and the different channels will then represent different tone patterns. 

In the changes from @volfan6415 there were now 4 preconfigure channels, but you could not add more channels for the same zones. So, in this change your could not have different tone patterns for different alarm events. I also felt that it was less likely that someone would be using all the preconfigured channels with the pro device (I will probably not). To avoid being too opinionated about the channels used, I left it to the user to configure the channels needed. I think it's a bit strange to configure channels,  but since this is core to how the binding works, I think it's better to be constant. In for a penny in for a pound.

Reply:
Thanks for the detailed explanation!